tests/ScannerTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace UNL\DWT;
```

Comment:
I would recommend against using the same namespace for src and test. It's easier to avoid collisions and accidental references to src classes. Recommend using `UNLTest\DWT` namespace.